modules/chapter1/pages/section2.adoc

```diff
@@ -1,3 +1,188 @@
-= Section 2
+= Section 2: OpenVINO Model Serving
 
-This is _Section 2_ of _Chapter 1_ in the *hello* quick course....
\ No newline at end of file
+In this section we will work in an excercise to deploy a model to an OpenVINO Serving Runtime.
+
+== Prepare MinIO
+
+https://min.io[MinIO] is a high-performance, S3 compatible object store. It is built for large scale AI/ML, data lake and database workloads. It is software-defined and runs on any cloud or on-premises infrastructure.
+
+We will need an S3 solution to share the model from training to deploy, in this exercise we will prepare MinIO to be such S3 solution.
+
+. In OpenShift, create a new namespace with the name **object-datastore**
++
+```shell
+oc new-project object-datastore
+```
+
+. Run the following yaml to install MinIO
++
+```shell
+curl https://raw.githubusercontent.com/RedHatQuickCourses/rhods-qc-apps/main/4.rhods-deploy/chapter2/minio.yml
+oc apply -f ./minio.yml -n object-datastore
+```
+
+. Get the route to the MinIO dashboard
++
+```shell
+oc get routes -n object-datastore | grep minio-ui | awk '{print $2}'
+```
++
+[INFO]
+====
+Use this route to navigate to the S3 dashboard using a browser. With the browser, you will be able to create buckets, upload files, and navigate the S3 contents.
+====
+
+. Get the route to the MinIO API
++
+```shell
+oc get routes -n object-datastore | grep minio-api | awk '{print $2}'
+```
++
+[INFO]
+====
+Use this route as the S3 API endpoint. Basically, this is the URL that we will use when creating a data connection to the S3 in RHODS.
+==== 
+
+== Training The Model
+We will use the iris dataset model for this excercise.
+
+. Using a JupyterLab workbench at RHODS, import the repository: https://github.com/RedHatQuickCourses/rhods-qc-apps.git
++
+[TIP]
+====
+It is recommended to use a workbench that was created with the **Standard Data Science** Notebook image.
+====
+
+. Make sure that the workbench environment serves the required python packages for the notebook to run, for this to happen, open a terminal and run the following command to verify that the packages are already installed:
++
+```shell
+  pip install -r /opt/app-root/src/rhods-qc-apps/4.rhods-deploy/chapter2/requirements.txt
+```
+
```

Comment:
We might want to suggest them to execute the command in the kernel process of the notebook:
```suggestion

[TIP]
====
You might also want to execute the preceding command in the notebook kernel by using the `%pip` syntax in the notebook. 
Alternatively, you can create a custom notebook image that includes the `skl2onnx` package.
====
```